index.md

```diff
@@ -54,7 +54,7 @@ sections:
       - title: "Language Spec"
         description: "Scala's formal language specification."
         icon: "fa fa-book"
-        link: http://scala-lang.org/files/archive/spec/2.12/
+        link: https://scala-lang.org/files/archive/spec/2.13/
```

Comment:
Could you apply same changes to `_ja/index.md` too?
That file is  Japanese translation of `index.md`